parse_test.go

```diff
@@ -19,9 +19,11 @@ func TestParse(t *testing.T) {
 			description: "GPP string with EU TCF V2",
 			gppString:   "DBABMA~CPXxRfAPXxRfAAfKABENB-CgAAAAAAAAAAYgAAAAAAAA",
 			expected: GppContainer{
-				Version:        1,
-				SectionStrings: []string{"DBABMA", "CPXxRfAPXxRfAAfKABENB-CgAAAAAAAAAAYgAAAAAAAA"},
-				Sectiontypes:   []constants.SectionID{3, 2},
+				Version: 1,
+				// SectionStrings: []string{"DBABMA", "CPXxRfAPXxRfAAfKABENB-CgAAAAAAAAAAYgAAAAAAAA"},
```

Comment:
Is the `SectionStrings` here no longer needed?

Reply:
Already removed, just need to push. :)